ttnn/cpp/ttnn/operations/experimental/reduction/fast_reduce_nc/fast_reduce_nc.hpp

```diff
@@ -0,0 +1,42 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "device/fast_reduce_nc_device_operation.hpp"
+#include "ttnn/run_operation.hpp"
+#include "ttnn/operations/core/core.hpp"
+#include "ttnn/tensor/tensor.hpp"
+
+namespace ttnn {
+namespace operations::experimental::reduction   {
+
+struct FastReduceNCOperation {
+    static ttnn::Tensor operator()(
+        uint8_t queue_id,
+        const ttnn::Tensor& input,
+        const std::vector<int32_t>& dims,
+        const std::optional<const Tensor> output,
+        const ttnn::MemoryConfig memory_config,
+        std::optional<const DeviceComputeKernelConfig> compute_kernel_config);
+
+    static ttnn::Tensor operator()(
+        const ttnn::Tensor& input,
+        const std::vector<int32_t>& dims,
+        const std::optional<const Tensor> output,
+        const ttnn::MemoryConfig memory_config,
+        std::optional<const DeviceComputeKernelConfig> compute_kernel_config);
+};
+
+}  // namespace operations::experimental::reduction
+
+namespace experimental::reduction {
+
+constexpr auto fast_reduce_nc = ttnn::register_operation_with_auto_launch_op<
+    "ttnn::experimental::reduction::fast_reduce_nc",
```

Comment:
```suggestion
    "ttnn::experimental::fast_reduce_nc",
```

Reply:
It is not critical, but good for consistency. 
Similarly to how we register other ops in ttnn, we register ttnn::experimental::subcategory::op... as ttnn::exerimental::op

Reply:
then need to adjust python calls too

Reply:
Thanks Artem. I changed the registration to experimental and updated python calls